controllers/propagator/replication.go

```diff
@@ -16,6 +16,8 @@ import (
 	"open-cluster-management.io/governance-policy-propagator/controllers/common"
 )
 
+const argoCDCompareOptionsAnnotation = "argocd.argoproj.io/compare-options"
+
 // equivalentReplicatedPolicies compares replicated policies. Returns true if they match.
 func equivalentReplicatedPolicies(plc1 *policiesv1.Policy, plc2 *policiesv1.Policy) bool {
```

Comment:
Does `equivalentReplicatedPolicies()` need to be updated also with this logic so that it doesn't try to keep replicating the policy?

Reply:
This is covered because `buildReplicatedPolicy` is called to get the "desired" replicated policy.